src/screens/Asl/components/AslTable/index.js

```diff
@@ -120,11 +116,21 @@ const AslTable = props => {
     }
 
     const handlePrevPage = () => {
-      setPageNumber(pageNumber-1);
+      let newPage = pageNumber-1;
+      if (newPage < 0) {
+        setPageNumber(0);
+      } else {
+        setPageNumber(newPage);
```

Comment:
Needs a math.ceil; in theory the number of pages could be reduced such that the previous page is no longer valid.